ui/apps/platform/cypress/integration/vulnerabilities/nodeCves/nodeCvesOverviewPage.test.ts

```diff
@@ -68,25 +84,135 @@ describe('Node CVEs - Overview Page', () => {
         assertAvailableFilters(expectedFilters);
     });
 
-    it('should link to the correct details pages', () => {
-        // clicking a CVE in the list should navigate to the correct CVE details page
-        // clicking a Node in the list should navigate to the correct Node details page
+    it('should link a CVE table row to the correct CVE detail page', () => {
+        // Having a CVE in CI is unreliable, so we mock the request and assert
+        // on the link construction instead of the content of the detail page.
+        mockOverviewNodeCveListRequest();
+        visitNodeCveOverviewPage();
+
+        cy.get('tbody tr td[data-label="CVE"] a')
+            .first()
+            .then(($link) => {
+                const linkHref = $link.attr('href');
+                const linkName = $link.text();
+                expect(linkHref).to.match(new RegExp(`.*/${linkName}$`));
+            });
+    });
+
+    it('should link a Node table row to the correct Node detail page', () => {
+        visitNodeCveOverviewPage();
+        cy.get(vulnSelectors.entityTypeToggleItem('Node')).click();
+
+        visitFirstNodeLinkFromTable().then((name) => {
+            cy.get('h1').contains(name);
```

Comment:
Aha, forward thinking to use `contains` method in case RegExp match is ever needed.

Reply:
I pretty much favor `.contains()` over `":contains()"` in all cases right now. I don't believe there is any downside to using the former.